packages/create-plugin/templates/scenes-app/src/components/SecretInput/SecretInput.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
```

Comment:
I think we can delete this in favour of using `import { SecretInput } from '@grafana/ui'` as scenes is only available in `9.5.0+`, right?